content_handler/runtime_holder.cc

```diff
@@ -40,29 +40,44 @@ RuntimeHolder::~RuntimeHolder() {
       }));
 }
 
-void RuntimeHolder::Init(mojo::ApplicationConnectorPtr connector) {
+void RuntimeHolder::Init(mojo::ApplicationConnectorPtr connector,
+                         std::vector<char> bundle) {
   FTL_DCHECK(!rasterizer_);
   rasterizer_.reset(new Rasterizer());
-  mojo::ConnectToService(connector.get(), "mojo:framebuffer",
-                         mojo::GetProxy(&framebuffer_provider_));
-  framebuffer_provider_->Create(ftl::MakeRunnable([self = GetWeakPtr()](
-      mojo::InterfaceHandle<mojo::Framebuffer> framebuffer,
-      mojo::FramebufferInfoPtr info) {
-    if (self)
-      self->DidCreateFramebuffer(std::move(framebuffer), std::move(info));
-  }));
+  InitRootBundle(std::move(bundle));
+
+  mojo::ConnectToService(connector.get(), "mojo:view_manager_service",
+                         mojo::GetProxy(&view_manager_));
 }
 
-void RuntimeHolder::Run(const std::string& script_uri,
-                        std::vector<char> bundle) {
-  InitRootBundle(std::move(bundle));
+void RuntimeHolder::CreateView(
+    const std::string& script_uri,
+    mojo::InterfaceRequest<mozart::ViewOwner> view_owner_request,
+    mojo::InterfaceRequest<mojo::ServiceProvider> services) {
+  if (view_listener_binding_.is_bound()) {
+    // TODO(jeffbrown): Support multiple view instances (or not).
```

Comment:
The intent is that there will be a one-to-one correspondence between  `RuntimeHolder` and Mozart view.  That means there will be a one-to-many relationship between `ApplicationImpl` and `RuntimeHolder`.  This code is fine for now, but that's the current plan.